### PR TITLE
Add staging repo build step

### PR DESCRIPTION
**Pull request by DavidFair**
Blocked by #7 

Changes the Github workflow to always push the image, but `master` will push to our prod image whilst any PRs will push to a new staging repo we can test against.

### CODE REVIEW

File: .github/workflows/build_images.yaml [hunk 1]
```diff
@@ -36,17 +36,27 @@ jobs:
 
       - name: Login to Harbor, if building master
         uses: docker/login-action@v2
-        if: ${{ github.ref == 'refs/heads/master' }}
```
Comment (ElliottKasoar) [inline .github/workflows/build_images.yaml:39]: Why was this removed? It seems to contradict the description.

Comment (DavidFair) [reply .github/workflows/build_images.yaml:39]: So we always login, then we either push to staging or prod based on the branch
Since the login happens down both paths we could remove this check

Comment (ElliottKasoar) [reply .github/workflows/build_images.yaml:39]: Is it worth getting rid of `if building master` in the name then?

Comment (DavidFair) [reply .github/workflows/build_images.yaml:39]: Done :+1:

**Review by ElliottKasoar — APPROVED** [review]
